spanner/spansql/parser_test.go

```diff
@@ -111,6 +111,20 @@ func TestParseQuery(t *testing.T) {
 				},
 			},
 		},
+		{`SELECT AVG(PointsScored) AS total_points, FirstName, LastName AS surname FROM PlayerStats GROUP BY FirstName, LastName`,
```

Comment:
this test case is really not needed.

Reply:
Removed.
https://github.com/googleapis/google-cloud-go/pull/3286/commits/e514c20b4199458ab35272eebc11cf0bd0f05efb